_inc/client/state/site/actions.js

```diff
@@ -62,6 +65,28 @@ export const fetchSiteBenefits = () => {
 	};
 };
 
+export const fetchProducts = () => {
+	return dispatch => {
+		dispatch( {
+			type: JETPACK_PRODUCTS_FETCH,
+		} );
+		return restApi
+			.fetchProducts()
+			.then( products => {
+				dispatch( {
+					type: JETPACK_PRODUCTS_FETCH_RECEIVE,
+					products: products,
```

Comment:
Nit: could we simplify those?

```suggestion
					products
```